x-pack/plugins/monitoring/server/cluster_alerts/alerts_clusters_aggregation.test.js

```diff
@@ -64,7 +64,8 @@ const mockQueryResult = {
   },
 };
 
-describe('Alerts Clusters Aggregation', () => {
+// TODO: tests were not running and are not up to date.
```

Comment:
Mind creating a follow-up issue for this?

Reply:
@elastic/stack-monitoring do you want to be the ones creating the issue to follow up on those `to-dos`? Otherwise I can create it